WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -445,6 +470,23 @@ public void onEventMainThread(NotificationEvents.NotificationsChanged event) {
         mTabLayout.checkNoteBadge();
     }
 
+    @SuppressWarnings("unused")
+    public void onEventMainThread(ConnectionChangeReceiver.ConnectionChangeEvent event) {
+        updateConnectionBar(event.isConnected());
+    }
+
+    private void checkConnection() {
+        updateConnectionBar(NetworkUtils.isNetworkAvailable(this));
+    }
+
+    private void updateConnectionBar(boolean isConnected) {
+        if (isConnected && mConnectionBar.getVisibility() == View.VISIBLE) {
+            AniUtils.animateBottomBar(mConnectionBar, false);
```

Comment:
> When we're animating the connection bar can we apply the animation to the FAB as well?

That would be a nice touch, but it's much trickier than it probably sounds. The FAB is on the "My Site" tab fragment and the connection bar is below the main activity's view pager which hosts the tab fragments. So as soon as the connection bar is made visible it "pushes" up the view pager, causing the jump you're seeing.

I could prevent this by having the connection bar overlay the view pager, but that means the bottom portion of each tab is obscured, which I think is a much worse problem.